tests_unit/test_message.py

```diff
@@ -77,3 +81,25 @@ def test_should_raise_error_with_wrong_message(self):
             validation(self.rbmq_message)
         with self.assertRaises(ValidationException):
             validation.validate(self.rbmq_message)
+
+    def test_should_raise_error_with_invalid_json(self):
+        def deserialize(body: bytes):
+            json.loads(body.decode())
+
+        rbmq_message = RabbitMQMessage(
+            1,
+            AMQPMessage(
+                connection=None,
+                channel=None,
+                queue_name=None,
+                serialized_data=bytes('{"invalid": "json"', "utf-8"),
+                delivery_tag=None,
+                envelope=None,
+                properties=None,
+                deserialization_method=deserialize,
+                queue=None
+            ),
+        )
+        validation = MessageValidation()
+        with self.assertRaises(ValidationException):
+            validation.validate(rbmq_message)
```

Comment:
Esse teste vai passar se o json for inválido ou tiver conteúdo inválido (for um json válido mas faltar um campo obrigatório no json_schema). Seria bom ter uma forma de separar os dois casos no teste.

O [pytest](https://docs.pytest.org/en/6.2.x/assert.html) tem a opção de olhar  o conteúdo da exceção. Pela mensagem de texto daria para ver a diferença.

Reply:
Acho que nesse caso não teria problema, já que eu estou instanciando a classe `MessageValidation` sem explicitar o schema que, por padrão é `{}`.

Reply:
Mas toma cuidado que o MessageValidation é uma variável de ambiente, esse comportamento pode mudar no futuro